unstructured_inference/inference/layout.py

```diff
@@ -143,26 +143,46 @@ def from_image_file(
         try:
             image = Image.open(filename)
             format = image.format
-            image = image.convert("RGB")
+            if format == "TIFF":
+                images = [im.convert("RGB") for i, im in enumerate(ImageSequence.Iterator(image))]
+            else:
+                image = image.convert("RGB")
```

Comment:
I think instead of handling `TIFF` as a separate case below, you could populate `images` with a singleton here:
```suggestion
                images = [image.convert("RGB")]
```
... and then below iterate over `images`.

Reply:
Fixed!